tests/html/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Test HTML file</title>
+    </head>
+    <body>
+        <h1>Hi there</h1>
+    </body>
+</html>
```

Comment:
nit: this needs a newline at EOF or it will break `cat`.
```suggestion
</html>
```